src/layers/python.rs

```diff
@@ -292,10 +292,6 @@ fn cache_invalidation_reasons(
 
 /// Environment variables that will be set by this layer.
 fn generate_layer_env(layer_path: &Path, python_version: &PythonVersion) -> LayerEnv {
-    // Several of the env vars below are technically build-time only vars, however, we use
-    // `Scope::All` instead of `Scope::Build` to reduce confusion if pip install commands
```

Comment:
Can `pip install` command be used at runtime?

Reply:
Yes, but the other headers related env vars aren't set either, and the headers aren't available.

Reply:
Yes, all CNB layers are writeable. xref:
https://github.com/heroku/base-images/issues/268